### DISCUSSION TITLE
[Suggestion] Add camera zoom on Mouse Wheel

### DISCUSSION THREAD

**Discussion by Showdoze** [FakeFishGames/Barotrauma#7134] [Ideas and suggestions]
Just a suggestion, will be very helpful for people to pick up small items and modders to take close-up pics

**Comment by StruggleDuck** [comment]
I like the idea behind this, but I feel that a better alternative for picking up small items would be to have some kind of button that switches between the items you are hovering over, and to possibly make some of the smaller items have bigger selection radiuses, or to even change the way that items look when they are dropped, i.e. shotgun shells or revolver ammo could be dropped in stacks, with the appearance of a small ammo box.
